content/blog/2023/08/camel4-whatsnew/index.md

```diff
@@ -0,0 +1,218 @@
+
+This release also comes with our first basic (and very limited) support for Spring Boot native (AOT compilation).
+See more in the [aot-basic](https://github.com/apache/camel-spring-boot-examples/tree/main/aot-basic) example.
+
+### 5) Camel Quarkus
+
+The Camel Quarkus project is working on a new release with Camel 4 support, expected later this month.
+Stay tuned for their release announcements.
+
+### 6) Camel Main
+
+For users who just want to run Camel as a standalone application, we have `camel-main` which is constantly being improved.
+The core in `camel-main` is reused by Camel Spring Boot, Camel Quarkus, and Camel JBang as well.
+
+The inclusion of an embedded HTTP server is now made easy with the new `camel-platform-http-main` module.
+See more in the [camel-main](https://github.com/apache/camel-examples/tree/main/examples/main) example.
+
+### 7) Camel JBang
+
+We made a lot of improvements to Camel JBang. The `camel` CLI can now easily run with different Camel versions, for example:
+
+    camel run foo.yaml --camel-version=3.21.0
+    camel run foo.yaml --camel-version=3.20.6
+
+This is very handy when, for example, you need to trouble-shoot why _something_ started failed. For instance, in occasions when
+it works on version X but not on version Y, now you can quickly try to find out which version in between that started failing.
+
+You can also specify that `camel` CLI should use a specific Camel version by default. For example, if you have a newer version of
+Camel JBang installed, but must develop and use an older release:
+
+    camel version set 3.20.6
+
+We added the following new commands:
+
+- `camel config`: you can use to set custom user configuration
+- `camel log`: you can use to show logs of your running Camel integrations (can show logs for 1 or more Camel apps)
+- `camel trace`: you can use to show message tracing of your running Camel integrations (can show logs for 1 or more Camel apps)
+- `camel cmd send`: you can use to send messages to an existing running Camel integration
+- `camel get route-dump`: you can use to dump routes in XML or YAML format
+
+We migrated the maven resolver that Camel JBang uses to the latest version, and Camel JBang will now report more accurately whether
+a dependency was downloaded or resolved from a local maven repository. The `camel run` command also has a `--verbose` flag to output
+more details in dependency resolution that can help during troubleshooting.
+
+The `camel-main` runtime now supports exporting with Kubernetes manifest and build support, to make it easier to build
+container images that are ready to run on Kubernetes.
+
+We made many improvements to Camel JBang, making it a great way to experiment with Camel and a useful companion tool for traditional 
+Camel development.
+
+### 8) XML DSL with beans
+
+We have been working on to unify the YAML, XML and Java DSL, so that they have feature parity related to configuring beans. For example,
```

Comment:
```suggestion
We have been working on unifying the YAML, XML and Java DSL, so that they have feature parity related to configuring beans. For example,
```